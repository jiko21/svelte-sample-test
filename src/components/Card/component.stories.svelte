<script>
  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
  import Card from './component.svelte';
</script>

<Meta
  title="components/Card"
  component={Card}
  argTypes={{}}
/>

<Template let:args>
  <Card {...args}>
    aaaaaaaaaaaaaaaa
  </Card>
</Template>

<!-- More on args: https://storybook.js.org/docs/svelte/writing-stories/args -->
<Story
  name="Default"
  args={{}}
/>
