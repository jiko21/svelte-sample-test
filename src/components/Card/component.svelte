<div class="card">
  <slot></slot>
</div>
<style>
  .card {
      background-color: #eeeeee;
      border-radius: 4px;
      padding: 8px;
  }
</style>
