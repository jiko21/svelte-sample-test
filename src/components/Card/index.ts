import Card from './component.svelte';

export default Card;
