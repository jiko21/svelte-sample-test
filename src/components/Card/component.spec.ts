import {render} from '@testing-library/svelte'

import Card from './component.svelte';

describe('components/Card', () => {

  describe('disabled', () => {
    it('correctly render', () => {
      const component = render(Card);
      expect(component.container).toMatchSnapshot();
    });
  });
})
