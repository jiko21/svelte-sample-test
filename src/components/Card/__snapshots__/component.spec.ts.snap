// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Card disabled correctly render 1`] = `
<body>
  <div>
    <div
      class="card svelte-ab6tor"
    />
  </div>
</body>
`;
